src/ProjectTemplates/scripts/Test-Template.psm1

```diff
@@ -25,7 +25,7 @@ function Test-Template {
     }
 
     Write-Verbose "Patching Microsoft.AspNetCore.App";
-    $builtRuntime = Resolve-Path "$PSScriptRoot/../../../artifacts/installers/$Configuration/aspnetcore-runtime-*-dev-win-x64.zip";
+    $builtRuntime = Resolve-Path "$PSScriptRoot/../../../artifacts/packages/$Configuration/Shipping/aspnetcore-runtime-*-dev-win-x64.zip" | Where-Object { $_ -match "aspnetcore-runtime-[0-9.]+-dev-win-x64.zip" };
```

Comment:
Is this to avoid matching the `internal` zip?

Reply:
It was to avoid matching `aspnetcore-runtime-composite-10.0.0-dev-win-x64.zip`